osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -285,6 +286,9 @@ public enum GlobalAction
         [LocalisableDescription(typeof(GlobalActionKeyBindingStrings), nameof(GlobalActionKeyBindingStrings.ToggleInGameInterface))]
         ToggleInGameInterface,
 
+        [LocalisableDescription(typeof(GlobalActionKeyBindingStrings), nameof(GlobalActionKeyBindingStrings.ToggleInGameLeaderboard))]
+        ToggleInGameLeaderboard,
```

Comment:
This cannot be put in the middle of the enum like that. The backing storage uses the numerical value of each enum value for storage purposes. Was this tested at all?

Reply:
Oh! I hadn't thought about that. It would probably make more sense to move it down to the bottom of the enum. I can get on that

Reply:
Do you know of a way I could test to see if this would break anything else?

Reply:
Not really.

Reply:
![image](https://github.com/ppy/osu/assets/32515172/215db38a-8660-41ed-8c63-e36c10385990)

Moved the value to the bottom of the enum table. It isn't the most elegant solution, but other than specifying the integer values I'm not sure what else I could do.

Reply:
Setting up a new client on the master branch, then switching over to the leaderboard-toggle branch causes both the `toggle chat` and `leaderboard` binds to be tab. 

This isn't a huge issue though, since you can't focus the chat if actively playing a multiplayer game. 

Of note, the multiplayer lobby leaderboard doesn't appear to change its visibility based on this setting.

Reply:
> Setting up a new client on the master branch, then switching over to the leaderboard-toggle branch causes both the toggle chat and leaderboard binds to be tab

It's kind of an issue. In the past this was solved by appropriate realm migrations (https://github.com/ppy/osu/commit/8d21f0b04b430e9c0943213a64b59cdc71ea3e2e).

But given that toggle chat and leaderboard don't clash even when they're both tab, maybe it's fine to leave both as tab? Although invariably people will ask for toggleable leaderboard in multiplayer even though that's not possible even in stable (and for a reason).

Reply:
Oh, that is exactly what I was looking for. 

I believe itd be fine rn if both are bound to tab, but having a mini-migration in place would probably make sense. 

Hopefully I can get that system up and running real quick

Reply:
I used the realm migration to re-set the bind for toggle chat. 

If someone has already changed the bind tho, it will leave it as is